bitcoin/src/blockdata/transaction.rs

```diff
@@ -981,10 +981,19 @@ impl Transaction {
     /// transaction. It is impossible to check if the transaction is first in the block, so this
     /// function checks the structure of the transaction instead - the previous output must be
     /// all-zeros (creates satoshis "out of thin air").
-    pub fn is_coin_base(&self) -> bool {
+    #[doc(alias = "is_coin_base")] // method previously had this name
```

Comment:
Interesting, is the idea to remove this when we remove the deprecated function? I've never used this attribute myself.

Reply:
I'd keep it for as long as we can tolerate it. Anyone might remember the old name and search for it. It doesn't hurt I guess, it only makes it show up in search when you search for that name.

Reply:
Yeah, I wouldn't mind keeping it forever.